src/qt/window.cpp

```diff
@@ -335,6 +361,11 @@ bool wxWindowQt::Create( wxWindowQt * parent, wxWindowID id, const wxPoint & pos
                 QtSetScrollBar( wxHORIZONTAL );
             if ( style & wxVSCROLL )
                 QtSetScrollBar( wxVERTICAL );
+
+            //  Allow processing of QGesture events, if generated upstream.
+            m_qtWindow->setAttribute(Qt::WA_AcceptTouchEvents);
+            m_qtWindow->grabGesture(Qt::PanGesture);
+
```

Comment:
Do we really need to do it here in addition to doing it in `EnableTouchEvents()`?

Reply:
Good catch.  Those two lines may be safely deleted.